### PR TITLE
Fix cypress files to cypress version 10 (migration)

### PR DESCRIPTION
**Pull request by gadotroee**

### CODE REVIEW

File: acceptanceTests/cypress/e2e/testHelpers/TrafficHelper.js [hunk 1]
```diff
@@ -4,8 +4,6 @@ export const valueTabs = {
     none: null
 }
 
-const maxEntriesInDom = 13;
```
Comment (RoyUP9) [inline acceptanceTests/cypress/e2e/testHelpers/TrafficHelper.js:7]: was this not used?

Comment (gadotroee) [reply acceptanceTests/cypress/e2e/testHelpers/TrafficHelper.js:7]: correct.

File: acceptanceTests/Makefile [hunk 2]
```diff
@@ -1,2 +1,3 @@
 test: ## Run acceptance tests.
+	@npm install cypress@10.0.1 -y
```
Comment (RoyUP9) [inline acceptanceTests/Makefile:2]: can this be done in the setup file?

Comment (gadotroee) [reply acceptanceTests/Makefile:2]: no. it should be installed when we are in the directory of the acceptance test.
so it feels like this place is the best place to put it in

Comment (gadotroee) [reply acceptanceTests/Makefile:2]: (if it is installed before (like I tried) it fails with error of "module not found" when we are running the "npx cypress run.."

**Review by RoyUP9 — APPROVED** [review]

**Pull request merged**